packages/remirror__extension-events/src/events-extension.ts

```diff
@@ -204,6 +210,7 @@ export type HoverEventHandler = (
   event: MouseEvent,
   state: HoverEventHandlerState,
 ) => boolean | undefined | void;
+export type EditableEventHandler = (currentEditable: boolean) => boolean | undefined | void;
```

Comment:
I guess we don't need to return a boolean value here.

Reply:
> We don't need to return a boolean value here, as the framework doesn't check the return value for this handler.

sorry, I forgot to make changes here